test/parallel/test-internal-util-assertCrypto.js

```diff
@@ -0,0 +1,12 @@
+// Flags: --expose-internals
```

Comment:
FWIW, this test won't actually increase test coverage reported at coverage.nodejs.org unless it is run with Node without crypto. And then, it would lose coverage on the `else` condition.